v2/charts/azure-service-operator/templates/generated/apps_v1_deployment_azureserviceoperator-controller-manager.yaml

```diff
@@ -0,0 +1,203 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: azure-service-operator
+    app.kubernetes.io/version: v2.7.0
+    control-plane: controller-manager
+  name: azureserviceoperator-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        {{- if .Values.podAnnotations }}
+        {{ toYaml .Values.podAnnotations }}
+        {{- end }}
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+   {{- if or (eq .Values.multitenant.enable false) (eq .Values.azureOperatorMode "watchers") }}
+        aadpodidbinding: aso-manager-binding
+   {{- end }}
+        app.kubernetes.io/name: azure-service-operator
+        app.kubernetes.io/version: v2.7.0
+        control-plane: controller-manager
+    spec:
+      {{- with .Values.tolerations }}
```

Comment:
minor: Fix indentation here (the `- with`) now that it's handcrafted/maintained?

Reply:
Did you mean `{{ - with .Values.tolerations }}` or the yaml indentation level?

Reply:
This: `{{- if .Values.podAnnotations }}` and the `{{- end }}` don't match the indentation of the other ifs in this file, basically is what I was getting at.

Some other files have similar wonkiness.

Since these are now handcrafted rather than generated via script I basically am wondering if it's worth doing a pass through them and moving all of the `if`s to have no indentation at all (always left-hand justified)? Or making them all line up with the indentation of the thing they're surrounding, but I think that's a bit harder to read so probably no indentation at all is better?

I'm not sure which way is idiomatic Helm and I don't know that it matters all that much which we pick, but I think we should be consistent, either:
```
   {{- if not .Values.multitenant.enable }}
  name: azureserviceoperator-manager-rolebinding
   {{ else }}
```

OR

```
{{- if not .Values.multitenant.enable }}
  name: azureserviceoperator-manager-rolebinding
{{ else }}
```

I have a slight preference towards the second but have a strong preference towards consistency of some kind (even if it's the first).

Reply:
Yep, sounds good. I can have a take on fixing the indentations in the following PR with #3837 if that's alright?

Reply:
sgtm